notebooks/wav2lip/ov_inference.py

```diff
@@ -0,0 +1,554 @@
+from os import listdir, path
+import numpy as np
+import scipy
+import cv2
+import os
+import sys
+import argparse
+from Wav2Lip import audio
+import json
+import subprocess
+import random
+import string
+from tqdm import tqdm
+from glob import glob
+import torch
+from Wav2Lip.models import Wav2Lip
+import platform
+import openvino as ov
+from pathlib import Path
+import torch
+from torch.utils.model_zoo import load_url
+from enum import Enum
+import torch.nn.functional as F
+from bbox import *
+
+try:
+    import urllib.request as request_file
+except BaseException:
+    import urllib as request_file
+
+
+device = "cpu"
+
+
+def get_smoothened_boxes(boxes, T):
+    for i in range(len(boxes)):
+        if i + T > len(boxes):
+            window = boxes[len(boxes) - T :]
+        else:
+            window = boxes[i : i + T]
+        boxes[i] = np.mean(window, axis=0)
+    return boxes
+
+
+def detect(net, img, device):
+    img = img - np.array([104, 117, 123])
+    img = img.transpose(2, 0, 1)
+    img = img.reshape((1,) + img.shape)
+
+    img = torch.from_numpy(img).float().to(device)
+    BB, CC, HH, WW = img.size()
+
+    results = net({"x": img.numpy()})
```

Comment:
slightly strange manipulation from numpy to torch and back without any actions between, ov model accepts torch tensors as well as numpy as input

Reply:
converting to numpy is removed